package.json

```diff
@@ -24,11 +24,11 @@
     "@polkadot/util": "^10.0.2",
     "@polkadot/util-crypto": "^10.0.2",
     "@prosopo/api": "^0.1.10",
-    "@prosopo/contract": "^0.1.10",
-    "@prosopo/datasets": "^0.1.10",
     "axios": "^0.27.2"
   },
   "devDependencies": {
+    "@prosopo/contract": "^0.1.10",
```

Comment:
shouldn't these be dependencies rather than dev deps?

Reply:
Yes, good spot!